lib/ansible/utils/unsafe_proxy.py

```diff
@@ -369,3 +372,20 @@ def to_unsafe_text(*args, **kwargs):
 
 def _is_unsafe(obj):
     return getattr(obj, '__UNSAFE__', False) is True
+
+
+def _intern(string):
+    """This is a monkey patch for ``sys.intern`` that will strip
+    the unsafe wrapper prior to interning the string.
+
+    This will not exist in future versions.
+    """
+    if isinstance(string, AnsibleUnsafeText):
+        string = string._strip_unsafe()
+    return _sys_intern(string)
+
+
+if sys.intern is _intern:
```

Comment:
This doesn't cover the reload case- if this module is reloaded, the new copy of `_intern` is different.

Reply:
Ok, I can switch it to roughly what you recommending comparing against `types.BuiltinFunctionType`.  One moment...